pkgs/os-specific/linux/fuse/common.nix

```diff
@@ -54,13 +54,14 @@ in stdenv.mkDerivation rec {
     # $PATH, so it should also work on non-NixOS systems.
     export NIX_CFLAGS_COMPILE="-DFUSERMOUNT_DIR=\"/run/wrappers/bin\""
 
-    sed -e 's@/bin/@${util-linux}/bin/@g' -i lib/mount_util.c
+    substituteInPlace lib/mount_util.c --replace /bin/ ${util-linux}/bin/
     '' + (if isFuse3 then ''
       # The configure phase will delete these files (temporary workaround for
       # ./fuse3-install_man.patch)
       install -D -m444 doc/fusermount3.1 $out/share/man/man1/fusermount3.1
       install -D -m444 doc/mount.fuse3.8 $out/share/man/man8/mount.fuse3.8
     '' else ''
+      substituteInPlace util/mount.fuse.c --replace '"su"' '"${shadow.su}/bin/su"'
```

Comment:
Not sure if this is really a good idea. Do you know why `/run/wrappers/bin` isn't in your `$PATH`? Given that `mount.fuse` should only be relevant for `root` this should be fine but usually we'd want the setuid `su` here.

In any case I feel like the commit message should explain the reasoning for this.